plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -357,19 +357,20 @@ public void testAnalyzeSomeColumns()
                         "('name', null, null, 0.0, null, null, null)," +
                         "(null, null, null, null, 50.0, null, null)");
 
+        String expectedFullStats = "VALUES " +
+                "('nationkey', null, 50.0, 0.0, null, 0, 49)," +
+                "('regionkey', null, 10.0, 0.0, null, 0, 9)," +
+                "('comment', 3764.0, 50.0, 0.0, null, null, null)," +
+                "('name', 379.0, 50.0, 0.0, null, null, null)," +
+                "(null, null, null, null, 50.0, null, null)";
+        assertUpdate(format("ANALYZE %s WITH(force_recalculate_statistics = true)", tableName));
```

Comment:
Let's actually showcase that `recalculate_statistics` does work with the columns which, when used previously without this option in the test, was causing a statement failure.

```
ANALYZE %s WITH(force_recalculate_statistics = true, columns = ARRAY['nationkey', 'regionkey', 'name'])
```

Reply:
This can be a separate test case, here i want to show that using `full_refresh` or however we will name it results in exactly the same stats as you get after dropping existing stats